x/gov/keeper/proposal.go

```diff
@@ -43,6 +43,9 @@ func (keeper Keeper) SubmitProposal(ctx context.Context, messages []sdk.Msg, met
 	for _, msg := range messages {
 		msgsStr += fmt.Sprintf(",%s", sdk.MsgTypeURL(msg))
 
+		// TODO HV2: sdk.Msg has 40k+ implementations that theoretically can pass the proposal
+		//  out of those, only 272 have `HasValidateBasic` function that could fail the proposal at `ValidateBasic`
```

Comment:
Where are you quoting this from?

Reply:
Yeah this is just a note for out discussion today. Will show you what the issue is with this.

Reply:
Outdated, will resolve.